controls/stig_rhel9/SRG-OS-000074-GPOS-00042.yml

```diff
@@ -2,8 +2,7 @@ controls:
     -   id: SRG-OS-000074-GPOS-00042
         levels:
             - medium
-        title: The operating system must transmit only encrypted representations of passwords.
+        title: {{{ full_name }}} must transmit only encrypted representations of passwords.
         rules:
             - package_vsftpd_removed
-            - sshd_allow_only_protocol2
```

Comment:
When you remove this rule ID from here, should you remove the SRG ID from its rule.yml?

Reply:
The reason why it was remove is because this option does nothing in RHEL9 (and the same could be said about RHEL8 as well).  Since [OpenSSH 7.6](https://www.openssh.com/txt/release-7.6) (Released 2017-10-03) OpenSSH has only support for SSH protocol version 2. So this SRG item is still valid for RHEL7.  Do we want to remove it from RHEL9 then?

Reply:
Removing it from RHEL 9 sounds logical to me.

Reply:
Since the rule is already in the RHEL 8 STIG, I think this is should be a RHEL9+ change.